spec/mongoid/criteria_spec.rb

```diff
@@ -3026,13 +3030,24 @@ class D
       end
 
       context 'when plucking the entire field' do
-
         let(:plucked) do
           Dictionary.all.pluck(:description)
         end
 
-        it 'returns all translations' do
-          expect(plucked.first).to eq({'en' => 'english-text', 'de' => 'deutsch-text'})
+        context "when legacy_pluck_distinct is set" do
+          config_override :legacy_pluck_distinct, true
+
+          it 'returns the non-demongoized translations' do
+            expect(plucked.first).to eq({"de"=>"deutsch-text", "en"=>"english-text"})
```

Comment:
Neil this is NOT correct.

This should be the result for `pluck(:description_transliations)`, not `pluck(:description)`

Reply:
Hey Johnny, I hear what you're saying and I had filed a ticket to correct this... I have a PR that's still in progress but does a lot of what you're asking for here: 
https://github.com/mongodb/mongoid/pull/5183

Reply:
Note: this is NOT finished yet